src/app/zap-templates/templates/app/MatterIDL.zapt

```diff
@@ -39,9 +39,7 @@
     nosubscribe {{!marker to place a space even with whitespace removal~}}
   {{~/unless~}}
   {{~!Removes spaces~}} attribute {{type}} {{asLowerCamelCase name~}}
-    {{~#if isList~}}
-      []
-    {{~/if}} = {{code}};
+    {{~#if isList~}} [] {{~/if}} = {{code}};
```

Comment:
Shouldn't we include maxLength here for string attributes?  Followup OK for that.

Reply:
Done. I keep forgetting that we have two very similar places for data.